src/main/java/com/hedera/hashgraph/sdk/Transaction.java

```diff
@@ -318,7 +338,7 @@ public final void executeForReceiptAsync(BiConsumer<Transaction, TransactionRece
      * while fetching a record.
      *
      * Call {@link #executeAsync(Consumer, Consumer)} then
-     * {@link #queryRecordAsync(Consumer, Consumer)} and handle the errors separately
+     * {@link #getRecordAsync(Consumer, Consumer)} and handle the errors separately
      * from each.
```

Comment:
Line 328 above, comment needs to be updated.

Reply:
Actually these `BiConsumer` overloads we decided to not carry over since the user shouldn't be doing everything in a single method chain anyway, so these comments can just be removed.